server/src-lib/Hasura/GraphQL/Context.hs

```diff
@@ -98,7 +117,11 @@ data ActionQuery (b :: Backend) v
   = AQQuery !(RQL.AnnActionExecution b v)
   | AQAsync !(RQL.AnnActionAsyncQuery b v)
 
-type RemoteField = (RQL.RemoteSchemaInfo, G.Field G.NoFragments G.Name)
+data RemoteField
```

Comment:
Yes, we can do it. But, I think this will only help with the `resolveRemoteField`, because the `RemoteField` type is scattered quite a bit around the code and implementing this change will require to change `RemoteField` to `RemoteField RemoteSchemaVariable` a lot of places. Which is why I don't think it's worth doing it.

Please let me know if you feel differently about it :)

Reply:
Done!